dac_adc.py

```diff
@@ -277,10 +284,12 @@ def buffer_ramp(self,c,dacPorts,adcPorts,ivoltages,fvoltages,steps,delay,nReadin
         voltages = []
         channels = []
         data = ''
+
+        dev.ramping(True)
```

Comment:
I did not place a yield because It's a return and not a returnValue, but I'm not sure